secrets.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_QA_URL=http://ec2-54-92-173-146.compute-1.amazonaws.com:1260
+export MONGODB_URI="mongodb://admin:foobar1@localhost:27017/test1"
+
+//blockstack key: youth already cream trade core frequent lumber quit engine recall chief divorce
```

Comment:
Please remove this

Reply:
whoops, don't know how that made it in there